engine/installation/linux/ubuntulinux.md

```diff
@@ -73,6 +74,7 @@ packages from the Docker repository:
     | Trusty 14.04 (LTS)  | `deb https://apt.dockerproject.org/repo ubuntu-trusty main` |
     | Wily 15.10          | `deb https://apt.dockerproject.org/repo ubuntu-wily main`   |
     | Xenial 16.04 (LTS)  | `deb https://apt.dockerproject.org/repo ubuntu-xenial main` |
+    | Yakkety 16.10 (LTS) | `deb https://apt.dockerproject.org/repo ubuntu-yakkety main` |
```

Comment:
Actually this should not be in master, because Ubuntu 16.10 is not yet supported (coming in docker 1.13) there are no packages available yet for docker 1.12.x (current release)